core/src/com/mygdx/hastypastry/models/Waffle.java

```diff
@@ -10,39 +12,38 @@
 
 public class Waffle {
     private Body wBody;
-    private Texture texture;
-    private Vector2 position;
+    private Sprite sprite;
+    private CircleShape shape;
```

Comment:
Trenger ikkje å holde på shape og bodyDef... Dei blir berre brukt for å definere bodien.